projects/packages/forms/src/modules/form-step/view.js

```diff
@@ -0,0 +1,21 @@
+import { getContext, store } from '@wordpress/interactivity';
+
+const NAMESPACE = 'jetpack/form';
```

Comment:
Package is "forms" so might use this also in plural?

```suggestion
const NAMESPACE = 'jetpack/forms';
```

Especially since later it's in plural here:

https://github.com/Automattic/jetpack/blob/b53e8885ca42b304f88a4cd2ae29384fc1c5ab03/projects/packages/forms/src/store/preview-store.js#L137